src/pages/PassportRedemption/Passport.tsx

```diff
@@ -69,7 +69,14 @@ const Passport = (props: Props) => {
         })
         .then((passportTickets) => {
           if (passportTickets.length > 0) {
-            setTickets(passportTickets);
+            const sortedTickets = passportTickets.sort((a, b) => {
+              const dateA = new Date(a.associated_with_contact_at);
+              const dateB = new Date(b.associated_with_contact_at);
+
+              return dateB.getTime() - dateA.getTime();
+            });
+
+            setTickets(sortedTickets.reverse());
```

Comment:
How come we're sorting then reversing? Possible to just reverse the sort? Also I think sorts like this should be handled in the backend

Reply:
for some reason, the sorting i do above only returns from latest to earliest, but then if i do the reverse in the sorting algorithm, it wouldnt give me the correct order. So i just did it like this